.eslintrc.js

```diff
@@ -71,13 +71,13 @@ module.exports = {
     'no-undef-init': 'error',
     'no-unused-expressions': 'error',
     'no-unused-labels': 'error',
-    'no-use-before-define': ['warn', {functions: false, classes: true}],
+    'no-use-before-define': 'off', // TS takes care of this one...
     'no-var': 'error',
     'prefer-const': 'error',
     'space-before-function-paren': 'off',
     'react/jsx-no-bind': 'off', // Keep perf implications in mind, but was giving too many warnings and hurting readability
     curly: 'error',
     radix: 'error',
-    'prettier/prettier': ['error', prettierConfig],
+    'prettier/prettier': ['warn', prettierConfig],
```

Comment:
I switched this to `warn`. I'd prefer to turn these rules off entirely since formatting is automatic anyways. I tried just running `yarn format:prettier`, but it matched more files than with just `yarn format` (which runs `eslint --fix`) so I get why this was enabled. The extra files were just some JSON files and MD files. I think that'd be okay to do and remove this rule entirely.

It just happens way too often then while I'm typing I get red squigglies and instantly think the issue is a type error. It happens while screen sharing as well. This wasn't a problem until I fixed this ESLint config file to be picked up by the ESLint plugin in VSCode: https://github.com/Workday/canvas-kit/pull/599